src/net/wooga/jenkins/pipeline/BuildVersion.groovy

```diff
@@ -3,6 +3,9 @@ package net.wooga.jenkins.pipeline
 import com.cloudbees.groovy.cps.NonCPS
 
 class BuildVersion {
+
+    static final String AUTO_UNITY_VERSION = "platform_version"

```

Comment:
can we name this `project_version`. Otherwise it could be confused with os version or whatever.

Reply:
Done.